web/packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -153,6 +155,26 @@ export class DocumentsService {
     };
   }
 
+  createGatewayKubeDocument(
+    opts: CreateGatewayKubeDocumentOpts
```

Comment:
I say let's just inline `CreateGatewayKubeDocumentOpts` here. Defining them in `web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts` is in my opinion an unnecessary pattern which I'm trying to get rid of. There's no use in moving the function signature outside of the definition of the function, especially if said signature is not reused in any way.